testIntegration/package.json

```diff
@@ -16,8 +16,7 @@
     "bundle:install": "npm run bundle && npm install",
     "bundle:clean:install": "npm run clean:all && npm run bundle:install",
     "test:prod": "npm run bundle:install && npm run test",
-    "test:staging": "COMMERCE_SDK_INPUT_DIR=testIntegration/stagingApis npm run test:prod",
-    "test:ci": "npm run test:prod && npm run test:staging",
+    "test:ci": "npm run test:prod",
```

Comment:
The tests already run against the apis downloaded to the apis directory so I don't see why we need to duplicate the download to the staging directory + test run.

Note: This may change in the near future if we start making a distinction between 'external' and 'internal' apis.